sample/build.gradle

```
@@ -54,13 +52,14 @@ dependencies {
     implementation "androidx.lifecycle:lifecycle-runtime-ktx:$ktxLifeCycle"
 
     // Compose
-    implementation "androidx.activity:activity-compose:$composeActivity"
-    implementation "androidx.compose.ui:ui:1.7.0-alpha08"
-    implementation "androidx.compose.ui:ui-tooling-preview:$compose"
+    api(platform("androidx.compose:compose-bom:$composeBom"))
```

app does not need to expose dependencies to higher-levels

```suggestion
    implementation(platform("androidx.compose:compose-bom:$composeBom"))
```